app/views/person_mailer/welcome_email.html.erb

```diff
@@ -24,7 +27,8 @@ Set Up Your Gmail Messages To Be From:  <%= @person.email %>
 </b>
 <ol>
   <li>
-    Log into your CMU-SV Gmail account at <a href="http://gmail.sv.cmu.edu/">http://gmail.sv.cmu.edu/</a>.
+    Log into your CMU-SV Gmail account at <a href="http://gmail.sv.cmu.edu/">http://gmail.sv.cmu.edu/</a>
+(a <a href="https://mail.google.com/a/west.cmu.edu/">re-direct</a>).
```

Comment:
Thanks for the feedback.

I tweaked the file a little.  I don't need to submit a new pull request,
right?
-

Albert Liu <albert.liu@sv.cmu.edu>, Senior IT Support Specialist
650-335-2819, Building 23, Room 210
CMU Silicon Valley

On Fri, Aug 14, 2015 at 8:39 PM, Todd Sedano <notifications@github.com>
wrote:

> In app/views/person_mailer/welcome_email.html.erb
> <https://github.com/professor/whiteboard/pull/274#discussion_r37132828>:
>
> > @@ -24,7 +27,8 @@ Set Up Your Gmail Messages To Be From:  <%= @person.email %>
> >  </b>
> >  <ol>
> >    <li>
> > -    Log into your CMU-SV Gmail account at <a href="http://gmail.sv.cmu.edu/">http://gmail.sv.cmu.edu/</a>.
> > +    Log into your CMU-SV Gmail account at <a href="http://gmail.sv.cmu.edu/">http://gmail.sv.cmu.edu/</a>
> > +(a <a href="https://mail.google.com/a/west.cmu.edu/">re-direct</a>).
>
> My suggestion would to be just have one url, not two
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/professor/whiteboard/pull/274/files#r37132828>.
>